libr/cons/d/monokai

```diff
@@ -1,15 +1,15 @@
-ec b0x00 rgb:222
+ec b0x00 rgb:555
 ec b0x7f rgb:fff
 ec b0xff rgb:eee
 ec other rgb:888
 ec bin rgb:888
-ec btext ef8d1a
-ec call rgb:000000 rgb:ef8d1a
+ec btext rgb:777
```

Comment:
rgb:ef8d1a

Reply:
Just update this because 777 is too ugly imho (not very different from bin or other) if you agree.
After that I can merge.